plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -85,6 +87,13 @@ private boolean validateLocalFields() {
       return false;
     }
     
+    File parent = getLocationPath().toFile();
+    File projectDirectory = new File(parent, getProjectName());
```

Comment:
So there's a bug here (and I meant to file it last night, except I accidentally installed an OS update and it was lost due to a reboot).  When 'default' is ticked (which is the default), then this code is correct: the project is created in the workspace.  But if the user has unticked 'default' then they specify a location for the *project* content, and the project name is unused in determining the directory name.  (It's a bit confusing, but I guess it's because there's no way other way for a user to say they want a directory different from the project name.)

I'd embed this knowledge into a `getNewProjectPath()` or something like that.